### DISCUSSION TITLE
Add types to AppContext

### DISCUSSION THREAD

**Discussion by rreeves8** [remix-run/remix#8602] [Q&A] [answered]
How do I add types to the load context between express and loader/action functions ? 
<img width="440" alt="Screenshot 2024-01-25 at 3 57 55 PM" src="https://github.com/remix-run/remix/assets/71095829/6dcfe2cf-c4e3-4676-8fca-36c7c447823a">
-> 
<img width="663" alt="Screenshot 2024-01-25 at 3 58 14 PM" src="https://github.com/remix-run/remix/assets/71095829/3f6b5c08-fce0-415e-b1ca-9aecb473208d">

Currently the context is unknown and I'd like to add a type to it.

**Comment by sergiodxa** [comment]
Check https://sergiodxa.com/tutorials/customize-remix-app-load-context-type

**Reply by rreeves8** [reply, marked as answer]
<img width="1326" alt="Screenshot 2024-01-25 at 4 26 37 PM" src="https://github.com/remix-run/remix/assets/71095829/c01706a4-e04e-463d-8331-9b8625e03de5">
Hey thanks for the reply, doesn't seem to be working for me ?

**Reply by rreeves8** [reply]
[artice](https://github.com/sergiodxa/sergiodxa.com/blob/main/content/tutorials/customize-remix-app-load-context-type.md)

**Reply by rreeves8** [reply]
This solution doesn't really work as I still dont have exact types on the context, is there a better method

**Reply by sergiodxa** [reply]
It should work now, the version on the GH repo is old, and I do that solution on my own blog and many other websites, so this must work

**Reply by rreeves8** [reply]
Great thanks!
